src/main/java/uk/gov/justice/hmpps/casenotes/controllers/CaseNoteController.java

```diff
@@ -30,24 +30,25 @@ public CaseNoteController(CaseNoteService caseNoteService) {
         this.caseNoteService = caseNoteService;
     }
 
-    @GetMapping("/{offenderIdentifier}")
-    @ApiOperation(value = "Returns list of case note for this offender",
-            response = CaseNote.class,
-            notes = "More Information Here")
-    @ApiResponses(value = {
-            @ApiResponse(code = 200, message = "Case Note list.", response = CaseNote.class, responseContainer = "List"),
-            @ApiResponse(code = 404, message = "No case notes where found for this offender", response = ErrorResponse.class)})
+    @RequestMapping(value = "/{offenderIdentifier}", method = RequestMethod.GET, consumes = "application/json", produces = "application/json")
+    @ResponseBody
+    @ApiOperation(value = "Retrieves a list of case notes",
+            nickname = "retrieve case notes")
+    @ApiResponses({
+            @ApiResponse(code = 404, message = "Case notes not found"),
```

Comment:
Not sure about this.  Wouldn't 404 be for when the offender is not found?  If an offender doesn't have any case notes wouldn't we want to just return a 200 with an empty list?

Reply:
Yes correct, this is a hangover from a merged endpoint!